extensions/wikia/UserLogin/js/UserLoginFacebook.js

```diff
@@ -111,10 +113,12 @@
 				// begin ajax call performance tracking
 				this.bucky.timer.start('loginCallbackAjax');
 
+				pageUrl = new QueryString();
+
 				// now check FB account (is it connected with Wikia account?)
 				$.nirvana.postJson('FacebookSignupController', 'index', {
-						returnto: encodeURIComponent(window.wgPageName),
-						returntoquery: encodeURIComponent(window.location.search.substring(1))
+						returnto: pageUrl.getVal('returnto'),
+						returntoquery: pageUrl.getVal('returntoquery')
```

Comment:
Are you sure this is the right change? Before, `returnto` was the title of the page, and `returntoquery` was the current querystring of the page. Now it's checking for two url parameters: `returnto` and `returntoquery`, which I don't think existed previously. Did something on the back end change that I'm missing?